lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
Wow. Didn't imagine that smartgit wouldn't allow me to set commit message wrap points. I'll endeavor to write the messages elsewhere and paste them.